src/Commands/Command.php

```diff
@@ -60,6 +60,13 @@
     protected $usage = 'Command usage';
 
     /**
+     * Show in Help
+     *
+     * @var bool
+     */
+     protected $showInHelp = true;
```

Comment:
For properties, we prefer snake case, i.e. `$show_in_help`

👍 for default being `true`